### PR TITLE
Updates:

### PR DESCRIPTION
**Pull request by Neketek**
1. Autocomplete
2. Changed distribution format to source dist
3. Wrapped cookiecutter into cli
4. Updated base image to latest

To enable autocomplete you need to:
1. ```make ssh```
2. ```make install```
3. ```exit```
4. ```make shell```
5. ```cot gen[tab]```

### CODE REVIEW

File: gen3-cli/cot/__init__.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Here we controll modules loading order
 # We can do it in command.__init__ but I guess this is a little bit more explicit
 import cot.command  # noqa
+import cot.command.generate
```
Comment (koriaf) [inline gen3-cli/cot/__init__.py]: no `noqa`?

File: gen3-cli/requirements/prod.txt [hunk 2]
```diff
@@ -1 +1,2 @@
 Click
+cookiecutter
```
Comment (koriaf) [inline gen3-cli/requirements/prod.txt]: let's freeze package versions already, at least major numbers there

File: gen3-cli/cot/__init__.py (hunk 1 above)
Comment (Neketek) [reply gen3-cli/cot/__init__.py]: That's interesting, but linter doesn't complain about this line.

File: gen3-cli/requirements/prod.txt (hunk 2 above)
Comment (Neketek) [reply gen3-cli/requirements/prod.txt]: Sure

**Pull request merged**